### PR TITLE
Fixed CI tests by excluding 1.0.0-beta3 and earlier

### PR DESCRIPTION
**Pull request by vlesierse**
This PR should fix the CI by excluding the 1.0.0-beta3 and earlier because of the breaking changes.

### CODE REVIEW

**Comment by vlesierse** [comment]
The core clr image is failing at the moment....
```
System.TypeInitializationException: The type initializer for 'Microsoft.Framework.Runtime.Common.CommandLine.AnsiConsole' threw an exception. ---> System.PlatformNotSupportedException: This platform does not support getting the current color.
   at System.ConsolePal.get_ForegroundColor()
   at Microsoft.Framework.Runtime.Common.CommandLine.AnsiConsole..ctor(TextWriter writer)
   at Microsoft.Framework.Runtime.Common.CommandLine.AnsiConsole..cctor()
   --- End of inner exception stack trace ---
   at Microsoft.Framework.PackageManager.Program.CreateReports(Boolean verbose, Boolean quiet)
   at Microsoft.Framework.PackageManager.Program.<>c__DisplayClass3_1.<<Main>b__2>d.MoveNext()
System.PlatformNotSupportedException: This platform does not support getting the current color.
   at System.ConsolePal.get_ForegroundColor()
   at Microsoft.Framework.Runtime.Common.CommandLine.AnsiConsole..ctor(TextWriter writer)
   at Microsoft.Framework.Runtime.Common.CommandLine.AnsiConsole..cctor()
lxc-start: The container failed to start.
lxc-start: Additional information can be obtained by setting the --logfile and --logpriority options.
```

**Comment by ahmetalpbalkan** [comment]
@vlesierse Thanks for the PR. Looks like the that sample is not supposed to run on Linux. Is it the command line app sample? If so, I thought we don't run that (because we can't validate the same way). If not, I can't see why Ansi color is required in a web app. I'll look more once I'm on desktop.

**Comment by ahmetalpbalkan** [comment]
@vlesierse it looks like `dnu restore` is just failing on Linux. Can you please report this to `aspnet/dnu`? I can see `1.0.0-beta4` is passing the tests fine so I think we may do another `grep -v 'coreclr-1.0.0-beta5-11624'` for now?

File: .ci/build-app-image.sh [hunk 1]
```diff
@@ -4,28 +4,28 @@ set -e 	# Exit immediately upon failure
...
-BRANCH=$4
+VERSION=$4
 
 echo "[CI] Injecting Dockerfile to project $TEST_APP..."
-cd  $SAMPLES_REPO/samples/$TEST_APP
-echo "[CI] Checkout branch $BRANCH"
-git checkout $BRANCH
+if [[ -d $SAMPLES_REPO/samples/$VERSION/$TEST_APP ]]; then
+	cd  $SAMPLES_REPO/samples/$VERSION/$TEST_APP
+else
```
Comment (ahmetalpbalkan) [inline .ci/build-app-image.sh]: minor suggestion: maybe another `-d ...` and `else` would fail with an error message?

File: 1.0.0-beta4/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,10 @@
-FROM mono:4
```
Comment (ahmetalpbalkan) [inline 1.0.0-beta4/Dockerfile]: can you undo these changes (I guess you need to rebase on top of master)

File: vs-1.0.0-beta4/Dockerfile [hunk 3]
```diff
@@ -1,36 +0,0 @@
-FROM mono:3.12
```
Comment (ahmetalpbalkan) [inline vs-1.0.0-beta4/Dockerfile]: can you undo changes this file (I guess you need to rebase on top of master)

File: vs-1.0.0-beta4/NuGet.Config [hunk 4]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (ahmetalpbalkan) [inline vs-1.0.0-beta4/NuGet.Config]: can you undo changes this file (I guess you need to rebase on top of master)

**Comment by vlesierse** [comment]
I'll make the changes and post an issue in the `aspnet/dnu`. Maybe an option like `--no-color` could be enough.  I understand that this is due to a limitation within the core clr.

**Comment by ahmetalpbalkan** [comment]
@vlesierse yep looks like that's just not implemented in PAL b/c ANSI terminals have no default font color or bg color like windows consoles. However, `dnu restore` crashing on linux for that reason is certainly not good.

**Comment by ahmetalpbalkan** [comment]
@eilon why is cla required again for him? What's going on here?

**Comment by Eilon** [comment]
We just changed the repos from MS Open Tech to .NET Foundation so there's a different CLA. The new CLA is simpler then the old one and so should only take a few moments to re-sign. Apologies for the extra bit of work!

**Comment by vlesierse** [comment]
Ok, signed the CLA on https://cla.dotnetfoundation.org

**Comment by Eilon** [comment]
Thanks @vlesierse !

**Comment by ahmetalpbalkan** [comment]
LGTM. Thanks.

**Comment by ahmetalpbalkan** [comment]
BTW I realized ANSI color issue with `dnu` is tracked at various places:

https://github.com/aspnet/dnx/issues/1768
https://github.com/aspnet/dnx/issues/1708
https://github.com/aspnet/Home/issues/559
https://github.com/aspnet/dnx/issues/1751
https://github.com/aspnet/dnx/pull/1724
https://github.com/dotnet/coreclr/issues/882
https://github.com/aspnet/dnx/pull/1724

It's the right call to keep it out of CI for now.
